airbyte-integrations/connectors/source-redshift/src/main/resources/spec.json

```diff
@@ -9,29 +9,46 @@
     "properties": {
       "host": {
         "description": "Host Endpoint of the Redshift Cluster (must include the cluster-id, region and end with .redshift.amazonaws.com)",
-        "type": "string"
+        "type": "string",
+        "order": 1
       },
       "port": {
         "description": "Port of the database.",
         "type": "integer",
         "minimum": 0,
         "maximum": 65536,
         "default": 5439,
-        "examples": ["5439"]
+        "examples": ["5439"],
+        "order": 2
       },
       "database": {
         "description": "Name of the database.",
         "type": "string",
-        "examples": ["master"]
+        "examples": ["master"],
+        "order": 3
+      },
+      "schemas": {
+        "title": "Schemas",
+        "description": "The list of schemas to sync from. Defaults to user. Case sensitive.",
```

Comment:
what does "defaults to user" mean? Does that just mean we'll default to fetching all schemas?

Reply:
Updated, thanks